grammar.js

```diff
@@ -693,7 +699,7 @@ module.exports = grammar({
       $.package_type,
       $.hash_type,
       $.object_type,
-      parenthesize($._type_expression),
+      $.parenthesized_type_expression,
```

Comment:
What do you think about calling this `parenthesized_type` for consistency with the other kinds of types, which all end in `_type`? Alternatively we could change them all to end in `_type_expression`. The phrase 'type expression` always seemed a bit awkward to me, but I'd be ok with that as well.

Reply:
`parenthesized_type` is indeed more consistent. I never thought about the phrase 'type expression', I just copied it from the OCaml manual, but now I don't like it either. I replaced `type_expression` by `type` everywhere, and that's way better.